cpp/tests/test_abm_world.cpp

```diff
@@ -282,6 +283,35 @@ TEST(TestWorld, evolveMigration)
         EXPECT_EQ(home.get_number_persons(), 2);
         EXPECT_EQ(hospital.get_number_persons(), 1);
     }
+
+    // Test if a dead person can still migrate
+    {
+        auto t      = mio::abm::TimePoint(0) + mio::abm::hours(8);
```

Comment:
@DavidKerkmann Could you have a look on this test in particular to also validate?

Reply:
I added another check for this where the person started in Hospital and move to the Cemetery with increased severity.

Reply:
Could you please check my comments again?